test/test-functions

```diff
@@ -619,6 +620,7 @@ install_verity_minimal() {
         mkdir -p "$initdir/usr/lib/extension-release.d" "$initdir/usr/lib/systemd/system" "$initdir/opt"
         grep "^ID=" "$os_release" >"$initdir/usr/lib/extension-release.d/extension-release.app0"
         echo "${version_id}" >>"$initdir/usr/lib/extension-release.d/extension-release.app0"
+        echo "SYSEXT_SCOPE=portable" >>"$initdir/usr/lib/extension-release.d/extension-release.app0"
```

Comment:
to ensure we don't regress the default, I'd avoid adding this to this image (but ofc leave it in the other one below)